test/__snapshots__/v3_client/ApiClient.ts.snap

```diff
@@ -3,29 +3,29 @@ import type { OpenAPIConfig } from './core/OpenAPI';
 import { Interceptors } from './core/OpenAPI';
 import { FetchHttpRequest } from './core/FetchHttpRequest';
 
-import { CollectionFormatService } from './services/CollectionFormatService';
-import { ComplexService } from './services/ComplexService';
-import { DefaultService } from './services/DefaultService';
-import { DefaultsService } from './services/DefaultsService';
-import { DeprecatedService } from './services/DeprecatedService';
-import { DescriptionsService } from './services/DescriptionsService';
-import { DuplicateService } from './services/DuplicateService';
-import { ErrorService } from './services/ErrorService';
-import { FileResponseService } from './services/FileResponseService';
-import { FormDataService } from './services/FormDataService';
-import { HeaderService } from './services/HeaderService';
-import { MultipartService } from './services/MultipartService';
-import { MultipleTags1Service } from './services/MultipleTags1Service';
-import { MultipleTags2Service } from './services/MultipleTags2Service';
-import { MultipleTags3Service } from './services/MultipleTags3Service';
-import { NoContentService } from './services/NoContentService';
-import { NonAsciiÆøåÆøÅöôêÊService } from './services/NonAsciiÆøåÆøÅöôêÊService';
-import { ParametersService } from './services/ParametersService';
-import { RequestBodyService } from './services/RequestBodyService';
-import { ResponseService } from './services/ResponseService';
-import { SimpleService } from './services/SimpleService';
-import { TypesService } from './services/TypesService';
-import { UploadService } from './services/UploadService';
+import { CollectionFormatService } from './services';
```

Comment:
NOTE: in separate PR I will be updating to import all of these services in one import (rather than separate import statement for each.